src/Converters/JsonConverter.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Kros.Utils;
+using System;
+using System.Text.Json;
+
+namespace Kros.KORM.Converter
+{
+    /// <summary>
+    /// Converter for converting JSON value from DB to entity.
+    /// </summary>
+    /// <seealso cref="IConverter" />
+    public class JsonConverter : IConverter
+    {
+        private readonly Type _entityType;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="JsonConverter"/> class.
+        /// </summary>
+        /// <param name="entityType">Type of the entity.</param>
+        /// <exception cref="ArgumentNullException">The value of <paramref name="entityType"/> is null.</exception>
+        public JsonConverter(Type entityType)
```

Comment:
1. We can make the type generic and so do not need this parameter.
2. We should provide parameter (optional) for `JsonSerializerOptions`, in case someone want customized formatting.

Reply:
1. You are right. I don't know why I was under the impression that when I use non-generic interface, the class implementing it cannot be generic.
2. As both methods implement interface, I provided option to specify serialization options as constructor parameter.